rlclientlib/dedup.cc

```diff
@@ -152,7 +152,7 @@ float dedup_state::get_ewma_value() const {
 
 void dedup_state::update_ewma(float value)
 {
-  //This is racy, but update only reads and modifies a single value, so it won't lead to data corruption 
+  //This is racy, but update only reads and modifies a single value, so it won't lead to data corruption
```

Comment:
line damage. please avoid this as it makes diffs larger and makes future bisect / blame operations harder.

Reply:
no longer changing this